cocotb/drivers/amba.py

```diff
@@ -341,6 +342,7 @@ def _read_data(self):
                     "ARLEN   %d\n" % _arlen +
                     "ARSIZE  %d\n" % _arsize +
                     "ARBURST %d\n" % _arburst +
+                    "ARPROT %d\n" % _arprot +
```

Comment:
I would go the other way with this and remove the `_arprot` variable. And the same goes for `_awprot`. It's not particularly useful information, and isn't widely used, which is probably why it wasn't printed in the first place.

Reply:
Actually, my current project does/will use it (unfortunately, it's not verified using cocotb). Not sure if I would need to see it printed.